modular-docs-manual/content/topics/module_mod-docs-concept-examples.adoc

```diff
@@ -1,8 +1,8 @@
 [id="modular-docs-concept-examples"]
 = Concept module examples
 
-link:https://access.redhat.com/documentation/en-us/red_hat_process_automation_manager/7.8/html-single/creating_red_hat_process_automation_manager_business_applications_with_spring_boot/index#bus_app_business-applications[Red Hat Process Automation Manager business applications]
+link:https://access.redhat.com/documentation/en-us/red_hat_process_automation_manager/7.8/html/creating_red_hat_process_automation_manager_business_applications_with_spring_boot/bus_app_business-applications[Red Hat Process Automation Manager business applications]
```

Comment:
For longevity, would it be better to update these links to the latest releases? For example:
```suggestion
link:https://access.redhat.com/documentation/en-us/red_hat_process_automation_manager/7.13/html/creating_red_hat_process_automation_manager_business_applications_with_spring_boot/bus_app_business-applications[Red Hat Process Automation Manager business applications]
```